hotsos/cli.py

```diff
@@ -257,45 +257,41 @@ def cli(data_root, version, defs_path, templates_path, all_logs, debug,
             host.
         """  # noqa
 
-        minimal_mode = None
-        if short:
-            minimal_mode = 'short'
-        elif very_short:
-            minimal_mode = 'very-short'
+        _version = get_version()
+        if version:
+            print(_version)
+            return
 
-        HotSOSConfig.force_mode = force
         repo_info = get_repo_info()
         if repo_info:
             HotSOSConfig.repo_info = repo_info
```

Comment:
This could go in the config section below.

Reply:
ack